Kitodo/src/main/java/org/kitodo/helper/metadata/LegacyDocStructHelperInterface.java

```diff
@@ -0,0 +1,364 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.helper.metadata;
```

Comment:
Shouldn't this go to the legacy-package?

Reply:
Good catch! 👍